securedrop_client/gui/main.py

```diff
@@ -20,13 +20,13 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 import logging
-from PyQt5.QtWidgets import QMainWindow, QWidget, QVBoxLayout, QDesktopWidget, QStatusBar
+from PyQt5.QtWidgets import QMainWindow, QWidget, QHBoxLayout, QVBoxLayout, QDesktopWidget, \
```

Comment:
Totally a nit, but `autopep8` is at it again.

Reply:
Discussed with heartsucker, this is fine :)